frontend/src/index.tsx

```diff
@@ -1,32 +1,25 @@
 import { createRoot } from 'react-dom/client';
-import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { ToastContainer } from 'react-toastify';
 import { Layout } from 'containers';
 import RouteManager from 'containers/RouterManager';
-import { PersistGate } from 'redux-persist/integration/react';
-
-import store from './store/configureStore';
+import { Provider } from 'react-redux';
+import { setupStore } from 'store/store';
 
 import 'react-toastify/dist/ReactToastify.css';
 import 'styles/index.scss';
-import { QueryClient, QueryClientProvider } from 'react-query';
 
 const container = document.getElementById('root');
-// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 const root = createRoot(container!);
-const queryClient = new QueryClient()
+const store = setupStore();
+
 root.render(
-  <QueryClientProvider client={queryClient}>
-    <Provider store={store.store}>
-      <PersistGate loading={null} persistor={store.persistor}>
```

Comment:
It helps you simply save at local storage data you need(when it rewrite on readux soter), and in your actionCreator you may delete saving to ls code.